README.md

```diff
@@ -27,17 +27,25 @@ Platform customers.
 before you can create any virtual machines with Compute Engine. Find the menu icon at the top left, 
 then look for the *Billing* link in the navigation bar.
 
+1. Next you will want to install the
+[Cloud SDK](https://cloud.google.com/sdk/) and make sure you've
+successfully authenticated and set your default project as instructed.
+
 1. In order for `ansible` to create Compute Engine instances, you'll need a
 [Service Account](https://cloud.google.com/compute/docs/access/service-accounts#serviceaccount). 
 It's recommended that you create a new Service Account (don't use the default), called 'demo-ansible', for this demo.
 Make sure to create a new JSON formatted private key file for this Service Account. Also, note the *Email address* 
 of this Service Account (should be `demo-ansible@YOUR_PROJECT_ID.iam.gserviceaccount.com`) since 
 this will be required in the Ansible configuration files.
 
-1. Next you will want to install the
-[Cloud SDK](https://cloud.google.com/sdk/) and make sure you've
-successfully authenticated and set your default project as instructed.
-
+    Example creating instance and setting scopes/iam roles using cloud shell:
+    
+    $ gcloud compute instances create INSTANCE NAME --scopes demo-ansible@YOUR_PROJECT_ID.iam.gserviceaccount.com=cloud-platform
```

Comment:
If we're using cloud shell, do we actually need a separate instance?  Can't we just do everything in Cloud shell?